src/PhpSpec/Loader/ResourceLoader.php

```diff
@@ -61,6 +61,8 @@ public function load(string $locator = '', int $line = null): Suite
                 }
             }
             if (!class_exists($resource->getSpecClassname(), false)) {
+                $error = new \Error('Spec path does not match defined namespace.');
```

Comment:
I think this message is too specific; all we know here is that the spec was not found in the file - maybe we can report the expected FQCN and file path and be quite explicit?

Also not sure about `\Error` here - this can be a more specific exception